### ISSUE TITLE
Training question and request for gt

### ISSUE THREAD

**Issue by C-water** [aiff22/PyNET-Bokeh#9]
Hi, 
1. When I training model LEVEL-1, the system warned that memory out. Could you please tell me the size of RAM you used before?
2. I registered the competition account, but I could not find the groundtruth of the validation set and test set. Could you please tell me where I can download them?

All requests are for academic research.
